pkg/blocktestservice/setup.go

```diff
@@ -0,0 +1,197 @@
+// Package blocktestservice is used for creating the blocklist file
+package blocktestservice
```

Comment:
use git mv command to do renaming

Reply:
tried restoring using git mv but did not work out